Committees/Chapter Committees/CFEC/CFEC.adoc

```diff
@@ -0,0 +1,37 @@
+= Chapter Firearms Education Committee (CFEC)
+// Title of committee goes here
+:toc:
+
+== Composition
+// Use this space to describe who the committee is composed of
+The CFEC is composed of chapter members who meet the requrements listed below in the requirements section, and have applied for the committee. The committee is lead by the <<CFED.adoc#,Chapter Firearms Education Director (CFED)>>
+
+== Requirements
+// This section should describe the requirements that need to be met to join  this committee. For national or regional committees, include this section:
+// Joining this committee requires you fulfill the requirements listed here (link to relevant chapter committee director)
+To join the CFEC, you must complete the following PLTCOM courses:
+
+* PLTCOM Intro to Scientific Socialism
+* PLTCOM Intro to Firearms
+
+Course equivalencies may be accepted based on the judgement of the CFED and National Central Committee. No furthet experience with firearms is necessary.  
+
+Further, members must submit an application to join the committee. The application asks the following questions:
+
+* How much experience do you have with firearms
+* How many hours per week, on average, are you willing to commit to this committee
+* How many committees are you already a member of
+
+These questions are meant to assess a members ability to participate in the committee and to develop a training plan for a members growth. Importantly, members will **NOT** be excluded for lack of firearms experience.
```

Comment:
a members ability -> the applicant's ability
a members growth -> the further development of the applicant's firearms proficiency, with the aim of heightening their capacity to contribute.